shared/common-adapters/floating-box/relative-floating-box.desktop.tsx

```diff
@@ -334,10 +338,15 @@ export class RelativeFloatingBox extends React.PureComponent<
         ? this.popupNode.getBoundingClientRect()
         : {height: -1, width: -1}
       if (snapshot.width !== width || snapshot.height !== height) {
-        this._computeStyle(this.props.targetRect)
+        // need to defer this until it actually renders
```

Comment:
i think this is a race between the child updating and it not actually rendering yet. i'd use a layoutEffect or similar but this is an old class so i'm doing this for now. To avoid a jump i'm using a fade in which is likely nice anyways